Flick/Cells/ProfileSummaryTableViewCell.swift

```diff
@@ -53,27 +53,17 @@ class ProfileSummaryTableViewCell: UITableViewCell {
         setupConstraints()
     }
 
-    private func calculateFriendsPreviewWidth() -> Int {
-        // Calculate width of friends preview based on number of friends and spacing between cells
-        if friends.count == 0 { return 0 }
-        let numFriendsInPreview = min(friends.count, maxFriendsPreview) + 1
-        let fullFriendsWidth = numFriendsInPreview * 20
-        let overlapFriendsWidth = (numFriendsInPreview-1) * condensedCellSpacing * -1
-        let friendsPreviewWidth = fullFriendsWidth - overlapFriendsWidth
-        return friendsPreviewWidth
-    }
-
     private func calculateUserInfoViewWidth(friendsPreviewWidth: Int) -> Int {
         let padding = friends.count == 0 ? 0 : 20
         let userNameLabelWidth = Int(usernameLabel.frame.size.width)
         let userInfoViewWidth = userNameLabelWidth + padding + friendsPreviewWidth
         return userInfoViewWidth
     }
-    
+
     private func setupConstraints() {
         let padding = 20
-        let friendsPreviewWidth = calculateFriendsPreviewWidth()
-        let userInfoViewWidth = calculateUserInfoViewWidth(friendsPreviewWidth: friendsPreviewWidth)
+        let friendsPreviewWidth = friendsPreviewView.getUsersPreviewWidth()
+        let userInfoViewWidth = calculateUserInfoViewWidth(friendsPreviewWidth: Int(friendsPreviewWidth))
```

Comment:
Are we not allowed to use CGFloats? Sometimes the width might end up being a rounded number no? does snap kit only allow ints?

Reply:
oh I converted it to Int since the function was `calculateUserInfoViewWidth(friendsPreviewWidth: Int) -> Int`. I'll change it to CGFloats